robottelo/vm.py

```diff
@@ -269,10 +268,9 @@ def install_katello_agent(self):
         result = self.run('rpm -q katello-agent')
         if result.return_code != 0:
             raise VirtualMachineError('Failed to install katello-agent')
-        if bz_bug_is_open('1431747'):
-            gofer_start = self.run('service goferd start')
-            if gofer_start.return_code != 0:
-                raise VirtualMachineError('Failed to start katello-agent')
+        gofer_start = self.run('service goferd start')
```

Comment:
@oshtaier we had yesterday a discussion on IRC and Eric Helm confirmed that Katello agent is not expected to auto-start any more on 6.3